src/Agent.Plugins/PipelineCache/TarUtils.cs

```diff
@@ -183,10 +146,6 @@ public static class TarUtils
                 catch (Exception e)
                 {
                     actionOnFailure();
```

Comment:
is this supposed to run when exitCode != 0?

Reply:
Yup.

Reply:
👍